src/modules/verification/verification.service.ts

```diff
@@ -0,0 +1,97 @@
+import { BadRequestException, Injectable, Logger } from '@nestjs/common';
+import { ConfigurationService } from '../../configuration/configuration.service';
+
+const Twilio = require('twilio');
+
+@Injectable()
+export class VerificationService {
+  private readonly logger = new Logger(VerificationService.name);
+  private readonly expirationTime = 900;
+
+  private twilioClient;
+  private sendingPhoneNumber: string;
+  private appHash: string;
+
+  // TODO: Replace with Database, Memcache or Redis?
```

Comment:
for the beginning, our Postgres is probably enough? Otherwise we need to orchestrate another container. 
The load on the verification will be minimal.